src/test/java/seedu/address/logic/parser/CloseTutorialCommandParserTest.java

```diff
@@ -0,0 +1,88 @@
+package seedu.address.logic.parser;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TUTORIAL;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.CloseTutorialCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.tutorial.Tutorial;
+
+public class CloseTutorialCommandParserTest {
+
+    private CloseTutorialCommandParser parser;
+
+    @BeforeEach
+    public void setUp() {
+        parser = new CloseTutorialCommandParser();
+    }
+
+    @Test
+    public void parse_validInput_returnsCloseTutorialCommand() throws Exception {
+        String args = " tut/Math";
+        CloseTutorialCommand command = parser.parse(args);
+
+        // Check that the parser returns a CloseTutorialCommand with the correct parameters
+        assertEquals(new CloseTutorialCommand(new Tutorial("Math")), command);
+    }
+
+    @Test
+    public void parse_missingTutorialPrefix_throwsParseException() {
+        String args = "Math";
+        ParseException exception = assertThrows(ParseException.class, () -> parser.parse(args));
+
+        // Verify that the exception message contains the expected usage message
+        assertTrue(exception.getMessage().contains(CloseTutorialCommand.MESSAGE_USAGE));
```

Comment:
Could we use assertParseFailure() here?

Reply:
Yep we can, I will change it in my code and push it again, thanks for the spot!